_vulnerabilities.adoc

```diff
@@ -32,8 +32,8 @@ We only extend this mathematical notation with set union operator (i.e., `∪`)
 |Summary |JDBC appender is vulnerable to remote code execution in certain configurations
 |CVSS 3.x Score & Vector |6.6 MEDIUM (CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H)
 |Components affected |`log4j-core`
-|Versions affected |`[2.0-beta7, 2.3.2) ∪ [2.4, 2.12.4) ∪ [2.13.0, 2.17.1)`
-|Versions fixed |`2.3.2` (for Java 6), `2.12.4` (for Java 7), or `2.17.1` (for Java 8 and later)
+|Versions affected |`[2.0-beta7, 2.3.1) ∪ [2.4, 2.12.3) ∪ [2.13.0, 2.17.0)`
+|Versions fixed |`2.3.1` (for Java 6), `2.12.3` (for Java 7), or `2.17.0` (for Java 8 and later)
```

Comment:
1. Shouldn't this stay `2.3.2`, since it adds `isJndiJdbcEnabled()` to the `isJndiEnabled` in ae4dc0c1a6389ae895f60451cd87e97e7b9d85c1?
2. Same for `2.12.4` due to b3879958cb3b75eb16bf27e970538b6e4e130f1c
3. Same for `2.17.1` due to 05db5f9527254632b59aed2a1d78a32c5ab74f16

Reply:
[LOG4J2-3242 "Limit JNDI to the java protocol only"](https://issues.apache.org/jira/browse/LOG4J2-3242) was implemented one patch version before.

Regardless whether the behavior in [CVE-2021-44832](https://www.cve.org/CVERecord?id=CVE-2021-44832) is a vulnerability or not, in versions `2.3.1`, `2.12.3` and `2.17.0` it was not possible to use JNDI schemas other than `java:`.